### DISCUSSION TITLE
Updated example of using 1.0 model to transcribe audio?

### DISCUSSION THREAD

**Discussion by cdleong** [coqui-ai/STT#1988] [Q&A]
Hello! I'm trying to find an updated example of using the [latest model](https://coqui.ai/english/coqui/v1.0.0-huge-vocab) to transcribe, well, any .wav file would be fine. 

Places I've looked none of which seem to work: 
* https://stt.readthedocs.io/en/latest/Python-Examples.html complains about `RuntimeError: CreateModel failed with 'Error reading the proto buffer model file.' (0x3005)`, when I point it to the `model.tflite` file.
* https://github.com/coqui-ai/STT-examples/blob/r0.9/vad_transcriber/audioTranscript_cmd.py doesn't seem to work, and also is based on v0.9. It seems to be looking for .pbmm files and gives me an IndexError when it can't find one. `wavTranscriber.py", line 61`, called from audioTranscript_cmd.py", line 37.
* https://github.com/coqui-ai/STT/blob/main/transcribe.py doesn't seem to work with the model either, and keeps giving import errors, e.g. for `tensorflow.compat`. Tried `pip install -r requirements_transcribe.txt`, but if you look at that file ([right here](https://github.com/coqui-ai/STT/blob/main/requirements_transcribe.txt)) it literally just has `webrtcvad`. If you try just installing `pip install stt coqui_stt_training coqui_stt_ctcdecoder` you get this error: 
```
INFO: pip is looking at multiple versions of stt to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement tensorflow==1.15.4 (from coqui-stt-training) (from versions: 2.5.0rc0, 2.5.0rc1, 2.5.0rc2, 2.5.0rc3, 2.5.0, 2.5.1, 2.6.0rc0, 2.6.0rc1, 2.6.0rc2, 2.6.0)
ERROR: No matching distribution found for tensorflow==1.15.4
```

So to sum up: is there a clear, working, tested example, using the latest model, and with documented requirements?

**Comment by reuben** [comment]
>     * https://stt.readthedocs.io/en/latest/Python-Examples.html complains about `RuntimeError: CreateModel failed with 'Error reading the proto buffer model file.' (0x3005)`, when I point it to the `model.tflite` file.

This most likely mean you have an older version of the STT package which is trying to load a `.pbmm` model. If you make sure you're installing `stt==1.0.0` it should work fine.

> https://github.com/coqui-ai/STT-examples/blob/r0.9/vad_transcriber/audioTranscript_cmd.py doesn't seem to work, and also is based on v0.9. It seems to be looking for .pbmm files and gives me an IndexError when it can't find one. `wavTranscriber.py", line 61`, called from audioTranscript_cmd.py", line 37.

Yeah I'm updating the examples for 1.0 right now.

Basically as long as you have STT 1.0.0 installed you should be able to simply do:

```bash
$ pip install -U pip
$ pip install stt
$ stt --model model.tflite --scorer scorer.scorer --audio audio.wav
```

**Reply by cdleong** [reply]
Thank you for the prompt reply! 

Those steps worked well, I was able to successfully pass in a .wav file, created with ffmpeg based on Common Voice data. I also found a way to pipe it to a file. Since `stt` outputs the actual transcript on stdout and outputs the various status messages I think on stderr, the following seems to work nicely for creating a transcript file. 
```
stt --model path/to/model.tflite --scorer path/to/huge-vocabulary.scorer --audio path/to/clips/common_voice_en_whatever.wav > out.txt
```
